modules/ROOT/pages/copy-and-paste.adoc

```diff
@@ -0,0 +1,38 @@
+= Copy & paste options
+
+:title_nav: Copy and paste options
+:description: TinyMCE copy and paste
+
+== {productname} PowerPaste plugin
+
+The {productname} PowerPaste plugin is a premium feature that provides improved paste support, including support for Microsoft Word, Microsoft Excel and Google Docs content. It also includes the option to clean or merge the styles when pasting content into the editor.
+
+For information, see the xref:introduction-to-powerpaste.adoc[PowerPaste plugin].
+
+== Controlling how content is pasted
+
+These options affect the way HTML content is processed when pasted in to the editor.
```

Comment:
Are these a mix of core paste and powerpaste settings? Can't remember

Reply:
Some are, some aren't... I don't think we should mention anything here though as we have PowerPaste specific details in the PowerPaste plugin section of the docs.

Reply:
None of these are PowerPaste specific at least, right? I guess that's more what I was thinking - clarifying that these settings are part of TinyMCE core and that you don't need PowerPaste for them

Reply:
Yeah, none are PowerPaste specific.